observability/test/test_utils.py

```diff
@@ -40,6 +40,7 @@ class ObservabilityTestCase(str, Enum):
     TEST_CONFIGS_NO_CONFIG = 'test_configs_no_config'
     TEST_CONFIGS_INVALID_CONFIG = 'test_configs_invalid_config'
     TEST_CONFIGS_CUSTOM_LABELS = 'test_configs_custom_labels'
+    TEST_TRACE_DIFF_SETTING_ENDPOINTS = 'test_trace_diff_setting_endpoints'
```

Comment:
Can we do a TODO for testing traces across multiple hops?
Like Client A sends request to Server A, Server A acts as client and starts another request to Server B. In this way we can test trace propagation across binaries as well as in-process.

Reply:
It was tracked in another TODO list of post-GA features